kernel/dw1000.h

```diff
@@ -483,15 +484,17 @@ union dw1000_rx_time {
 #define DW1000_SAR_TEMP_MDEGC(sar, sar_23c) \
 	(23000 + ((100000 * ((sar) - (sar_23c)))) / 114)
 
-/* Maximum possible link quality indicator value */
+/* Maximum possible indicator values */
```

Comment:
Any reason for this wording change?  The old wording is still correct as far as I can tell.